code/modules/mob/living/simple_animal/friendly/reaver.dm

```diff
@@ -0,0 +1,49 @@
+/mob/living/simple_animal/reaver/baby
+	name = "baby reaver"
+	desc = "The result of SCC experimentation in exploitation of space fauna. While it looks like a dangerous whirl of talons, this one has been successfully tamed."
+	icon = 'icons/mob/npc/pets.dmi'
+	icon_state = "babyreaver"
+	item_state = "babyreaver"
+	icon_living = "babyreaver"
+	icon_dead = "babyreaver_dead"
+	organ_names = list("head", "chest", "tail", "left flipper", "right flipper")
+	meat_type = /obj/item/reagent_containers/food/snacks/fish/carpmeat
+	butchering_products = list(/obj/item/reagent_containers/food/snacks/fish/roe = 1)
+	meat_amount = 3
+	flying = TRUE
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_NOLIGHTING
+	density = FALSE
+	mob_size = MOB_SMALL
+	speed = 4
+	maxHealth = 50
+	health = 50
+
+	response_help = "pets the"
```

Comment:
Wouldn't this lead to "pets the the baby reaver"?

Reply:
This is the exact phrase used for space carp, including the enemy reaver. I do not believe this is proper/improper aware